lib/echo/text_to_speech.ex

```diff
@@ -39,4 +41,49 @@ defmodule Echo.TextToSpeech do
       end
     end)
   end
+
+  defp send_with_retry(pid, text, retries \\ 0)
```

Comment:
It's probably the best to leave retries out of the library and let the caller handle failures. "Let it crash"

Reply:
Good idea. Retries might be taking it a bit too far